pandas/_libs/tslibs/period.pyx

```diff
@@ -2483,9 +2483,11 @@ class Period(_Period):
     Parameters
     ----------
     value : Period or str, default None
-        The time period represented (e.g., '4Q2005').
+        The time period represented (e.g., '4Q2005'). This represents neither
+        the start or the end of the period, but rather the entire period itself.
     freq : str, default None
-        One of pandas period strings or corresponding objects.
+        One of pandas period strings or corresponding objects. Accepted
+        strings are listed under "Offset aliases" in the `timeseries` docs.
```

Comment:
```suggestion
        strings are listed in the :ref:`offset alias section <timeseries.offset_aliases>` in the user docs.
```

This will link directly to those aliases.